mon-pix/app/components/challenge-item-generic.js

```diff
@@ -66,6 +67,9 @@ export default class ChallengeItemGeneric extends Component {
       const timer = later(this, function() {
```

Comment:
**suggestion (non-blocking):** Peut-être stopper le tick quand `this.isTimeoutChallenge = true` pour éviter des surprises